Description: do not install init script in upstream
 debian/puavo-wlanap.init does it correctly
Author: Tuomas Räsänen <tuomasjjrasanen@opinsys.fi>

--- puavo-wlanap-0.3.0.orig/Makefile
+++ puavo-wlanap-0.3.0/Makefile
@@ -14,8 +14,6 @@ all :
 installdirs :
 	mkdir -p $(DESTDIR)$(sbindir)
 	mkdir -p $(DESTDIR)$(sysconfdir)/puavo-wlanap
-	mkdir -p $(DESTDIR)$(sysconfdir)/default
-	mkdir -p $(DESTDIR)$(sysconfdir)/init.d
 
 .PHONY : install
 install : installdirs
@@ -23,8 +21,6 @@ install : installdirs
 		sbin/*
 	$(INSTALL_DATA) -t $(DESTDIR)$(sysconfdir)/puavo-wlanap \
 		etc/puavo-wlanap/*
-	$(INSTALL_PROGRAM) -t $(DESTDIR)$(sysconfdir)/init.d \
-		etc/init.d/puavo-wlanap
 
 .PHONY : clean
 clean :
